src/main/java/uk/gov/justice/digital/delius/service/RequirementService.java

```diff
@@ -127,6 +127,7 @@ public LicenceConditions getLicenceConditionsByConvictionId(String crn, Long con
         return getRequirementsByConvictionId(crn, eventId)
             .getRequirements().stream()
             .filter(Requirement::getActive)
+            .filter(requirement -> !requirement.getSoftDeleted())
```

Comment:
isnt this technically a breaking change?

Reply:
This method is only used by our Referral Service that creates NSIs. I have done a find usages to confirm this.